### ISSUE TITLE
Incorrect fsf address

### ISSUE THREAD

**Issue by Hubbitus** [mooz/percol#77]
Hello.

I've package percol for Fedora and discower you have outdated file headers:
```
percol.x86_64: E: incorrect-fsf-address /usr/bin/percol
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/command.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/debug.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/view.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/tty.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/ansi.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/markup.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/display.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/action.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/lazyarray.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/__init__.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/info.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/model.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/actions.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/cli.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/key.py
percol.x86_64: E: incorrect-fsf-address /usr/lib/python3.4/site-packages/percol/finder.py
```

By our guidelines https://fedoraproject.org/wiki/Common_Rpmlint_issues#incorrect-fsf-address I shoul inform you.

**Comment by mooz** [comment]
Thanks for the report. I've corrected the file headers in version 0.1.1.

**Comment by Hubbitus** [comment]
Thank you.
